### DISCUSSION TITLE
Unable to download youtube video, all other videos work

### DISCUSSION THREAD

**Discussion by GGhi2** [aclap-dev/video-downloadhelper#1495] [Help and Q&A] [answered]
### Your question

I'm attempting to download this video, https://www.youtube.com/watch?v=y-EPpH5qUhc but nothing shows up in the extension, even tried using stacher. I at first thought that it was due to the video being up for only an hour, but when trying the same thing on other videos they show up. Any reason why I can't download it? 

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36 OPR/114.0.0.0
{
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 24
}

**Comment by Mattmjs** [comment, marked as answer]
I have detection without even playing it. But there is a problem in that your using Chrome. Downloading on Youtube is disabled on this
browser. You have to be on Firefox or Edge.

**Reply by GGhi2** [reply]
Didn't know this, thank you.
